.github/workflows/centos7/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -l
+set -x
+uname -a 
+cat /etc/issue
+yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
Why not use CentOS-7 default Clang and LLVM? They should be at version ~3.4? so test them seems to be logical thing to do (more on this below).

Reply:
EPEL does not contain a newer llvm, the default llvm 3.4 is used. EPEL is needed for a newer cmake, called  cmake3.